### ISSUE TITLE
Where are the Lua communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/lua#225]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Lua supported by one or more large organizations?
* Does Lua have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Lua? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Lua? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Lua? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Lua?
* Are there popular newsletters for Lua?
* Is Lua taught at programming bootcamps? (If so, what are some examples?)
* Is Lua taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Lua?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by WildYorkies** [comment]
The Lua community is extremely fragmented. But here are some helpful links:

http://lua.space/
https://luarocks.org/
http://torch.ch/
https://love2d.org/

I would recommend getting a retweet from https://twitter.com/LuaLang

**Comment by imolein** [comment]
There are also:
* Mailinglist: https://www.lua.org/lua-l.html
* IRC: #lua on freenode
* Telegram Group: https://t.me/LuaLang

**Comment by 225559** [comment]
The Lua [forum](https://luaforum.com) has a good community
